velox/docs/monthly-updates/january-2023.rst

```diff
@@ -0,0 +1,76 @@
+********************
+January 2023 Update
+********************
+
+Core Library
+============
+
+* Add generic writer which supports generic outputs in the simple function interface and thus unblocks authoring a large set of functions that operate on any type such as :func:`subscript`, :func:`array_concat`, :func:`array_intersect`.
```

Comment:
Perhaps, `Add support for authoring simple functions with generic results, e.g. :func:`subscript`,..`